docs/instructions.md

```diff
@@ -0,0 +1,164 @@
+# External Instruction Definitions
```

Comment:
This file is in good shape, except like before I think `prim_type` should be documented more thoroughly.